prost-derive/src/lib.rs

```diff
@@ -251,14 +251,290 @@ fn try_message(input: TokenStream) -> Result<TokenStream, Error> {
         #methods
     };
 
-    Ok(expanded.into())
+    Ok(expanded)
 }
 
-#[proc_macro_derive(Message, attributes(prost))]
-pub fn message(input: TokenStream) -> TokenStream {
-    try_message(input).unwrap()
+/// Takes a token stream such as `Clone, PartialEq, :: prost :: Message` and
+/// returns `Clone, PartialEq`.
```

Comment:
Why do we need to remove prost message ?

Reply:
`open_oneof` takes care of generating the 5 methods that `prost::Message` does. The `prost::Message` needs to be removed because it would shadow the implementations coming form `open_oneof`